app/services/ccms/applicant_add_requestor.rb

```diff
@@ -30,13 +36,13 @@ def client(xml)
     end
 
     def name(xml)
-      xml.__send__('ns4:Surname', 'Hurlock')
-      xml.__send__('ns4:FirstName', 'lenovo')
+      xml.__send__('ns4:Surname', @applicant.last_name)
+      xml.__send__('ns4:FirstName', @applicant.first_name)
     end
 
     # this is all mandatory: we don't hold any of this data except date of birth
     def personal_information(xml)
-      xml.__send__('ns5:DateOfBirth', '1969-01-01')
+      xml.__send__('ns5:DateOfBirth', @applicant.date_of_birth.strftime('%Y-%m-%d'))
```

Comment:
If this is the format CCMS likes dates I'd suggest adding the following to `config/initializers/date_formats.rb`

```ruby
ccms_formats = {
  cmms_date: '%Y-%m-%d'
}

Time::DATE_FORMATS.merge! ccms_formats
Date::DATE_FORMATS.merge! ccms_formats
```
Then here:
```ruby
xml.__send__('ns5:DateOfBirth', @applicant.date_of_birth.to_s(:ccms_date))
```